scripts/update_afc_ioc.sh

```diff
@@ -35,7 +35,7 @@ for crate in "${CRATES[@]}"; do
         echo 'epicsEnvSet(TOP, /opt/afc-epics-ioc)' >> iocBoot/iocutca/envPaths &&
         echo 'epicsEnvSet(AUTOSAVE_PATH, /var/opt/afc-epics-ioc)' >> iocBoot/iocutca/envPaths &&
         systemctl daemon-reload &&
-        systemctl --no-block restart afc-ioc@{1,2-1,4,5}") &> /tmp/update_afc_ioc_$crate.log &
+        systemctl --no-block restart afc-ioc@{1,2-1,4,5,6..12}") &> /tmp/update_afc_ioc_$crate.log &
```

Comment:
Did this syntax work for you?
I had listed them all out locally

Reply:
No, it does not. Sorry, I forgot to update it. When running with the copy from gca-jobs, I listed them all manually.